### PR TITLE
Updated broken links on ERC1155DelayedReveal.mdx

### PR DESCRIPTION
**Pull request by RobertAndre**

### CODE REVIEW

File: docs/onboarding/1 Solidity SDK/Base Contracts/ERC1155/ERC1155DelayedReveal.mdx [hunk 1]
```diff
@@ -11,7 +11,7 @@ import ViewContractCodeButton from "@components/ViewContractCodeButton";
 import "@thirdweb-dev/contracts/base/ERC1155DelayedReveal.sol";
 ```
 
-`ERC1155DelayedReveal` adds the [Delayed Reveal](/sdk/interacting-with-contracts/erc1155delayedreveal) feature to the
+`ERC1155DelayedReveal` adds the [Delayed Reveal](/glossary/delayed-reveal) feature to the
```
Comment (nachoiacovino) [inline docs/onboarding/1 Solidity SDK/Base Contracts/ERC1155/ERC1155DelayedReveal.mdx:14]:
```suggestion
`ERC1155DelayedReveal` adds the [Delayed Reveal](/solidity/extensions/delayedreveal) feature to the
```

**Review by nachoiacovino — CHANGES_REQUESTED** [review]
need to link to the extension

File: docs/onboarding/1 Solidity SDK/Base Contracts/ERC1155/ERC1155DelayedReveal.mdx (hunk 1 above)
Comment (nachoiacovino) [reply docs/onboarding/1 Solidity SDK/Base Contracts/ERC1155/ERC1155DelayedReveal.mdx:14]: okay no, makes sense what you did

**Review by nachoiacovino — APPROVED** [review]

File: docs/onboarding/1 Solidity SDK/Base Contracts/ERC1155/ERC1155DelayedReveal.mdx (hunk 1 above)
Comment (RobertAndre) [reply docs/onboarding/1 Solidity SDK/Base Contracts/ERC1155/ERC1155DelayedReveal.mdx:14]: Ok i wasn't sure, but it seemed like in that context it was trying to more info on the term. Im going to quickly go through the pages and make sure they link up.

Comment (nachoiacovino) [reply docs/onboarding/1 Solidity SDK/Base Contracts/ERC1155/ERC1155DelayedReveal.mdx:14]: yeah let's leave it with the glossary makes sense

**Pull request merged**